gradebook.py

```diff
@@ -19,8 +19,13 @@ def add_grade(self, instructor, course_id, student_id, grade):
         now = datetime.datetime.now()
 
         if student_id in self.grades[course_id]:
-            print_error("Error: Grade already exists. Use 'edit' instead.")
-            input("Press enter to continue.")
+            option = input("Grade already exists. Do you want to edit it? (Y/N): ")
+            if option.upper() == "Y":
+                self.grades[course_id][student_id]["grade"] = grade
+                print(f"\033[32mGrade updated for student {student_id}: {grade}\033[0m")
```

Comment:
Use print methods in util.py so that it can aligns with color themes.

Reply:
Check the else branch below, and the grade has a grade and a timestamp. Please use the same format.